src/Compilers/Core/Portable/Operations/ConversionKind.cs

```diff
@@ -31,6 +31,11 @@ public enum ConversionKind
         /// </summary>
         OperatorMethod = 0x5,
         /// <summary>
+        /// Conversion is defined by the underlying type, and is created when an interpolated
+        /// string expression is being converted to an IFormattable or FormattableString.
+        /// </summary>
+        InterpolatedString = 0x6,
```

Comment:
I remember that we have made this decision, but I am not certain we actually looked at the list of existing kinds in the enum, which could lead us to a different decision. Anyway, we can go with the change as it is, but please bring this to design meeting to confirm that this is what we want.